inc/Engine/Optimization/CSSTrait.php

```diff
@@ -35,6 +35,10 @@ public function rewrite_paths( $source, $target, $content ) {
 		 */
 		$target = apply_filters( 'rocket_css_asset_target_path', $target );
 
+		$content = $this->move( $this->get_converter( $source, $target ), $content, $source );
+
+		$content = $this->combine_imports( $content, $target );
```

Comment:
shouldn't the `combine_imports()` be before the `move()` call?

Reply:
Simply No, because I need to rewrite urls/paths as you some paths are relative to source file so I need to move them to the new url structure to be relative to our new target then be able to get their contents then you can see here

https://github.com/wp-media/wp-rocket/blob/43840d58d34a0c8a4f6ff74bcbfc0b34f1f06235/inc/Engine/Optimization/CSSTrait.php#L309

I'm calling the main method recursively to after combine_imports works for first time.
So don't worry about applying `move` after combining imports contents.